test/approvedTransfer.js

```diff
@@ -157,7 +157,7 @@ describe("Test Approved Transfer", function () {
         });
     });
 
-    describe("Transfer approved by smart-contract guardians", () => {
+    describe("Transfer approved by smart-contract guardians", () => { return;
```

Comment:
I think you forgot to remove a bunch of `return` statements in this file

Reply:
Ooops,  indeed.

Fixed.